test/api_stability/3.0.2/0002-NOAPI-DETAIL-MISC-Remove-top-level-scoring-scheme-de.patch

```diff
@@ -20,7 +20,7 @@ index ef141a7e9..7b0927215 100644
 -                                                                seqan3::align_cfg::method_global,
```

Comment:
please don't change patches, if they don't work they need to be changed differently :)